prow/jobs/images/Dockerfile.auto-generate-controllers

```diff
@@ -0,0 +1,41 @@
+FROM debian:buster-slim AS base
```

Comment:
note: we will want to switch this to an ECR-based one to avoid the issues with dockerhub limits...